jenkins/opensearch-dashboards/integ-test.jenkinsfile

```diff
@@ -230,73 +230,94 @@ pipeline {
                         // Must use local variable due to groovy for loop and closure scope
                         // Or the stage will be fixed to the last item in return when new stages are triggered here
                         // https://web.archive.org/web/20181121065904/http://blog.freeside.co/2013/03/29/groovy-gotcha-for-loops-and-closure-scope/
-                        def local_component = component.trim()
-                        def local_component_index = componentList.indexOf(local_component)
-                        def wait_seconds = local_component_index * 20
 
-                        echo "Adding Component: ${local_component}"
-                        componentTests["Run Integtest ${local_component}"] = {
-                            // Using scripted pipelines to trigger dynamic parallel stages
-                            timeout(time: 6, unit: 'HOURS') {
+                        // Changes to run OpenSearch-Dashboards ci-groups in parallel.
+                        // There are total 9 ci-groups
+                        if (component == "OpenSearch-Dashboards") {
+                          for (int i = 1; i <= 9; i++) {
```

Comment:
Can we dynamically assign this value as workflow params?

Reply:
Yes, can take it up in future improvements.

Reply:
Can you please create an issue for this? Also wondering if we can dynamically assign this value by fetching from test-manifest instead of hard coding to 9 here.
TL;DR:
```
ci-number = ${ci_number} <------------ parameter to this jenkins file 
if ci_number == null:
  ci-number = testManifest.OpenSearch-Dashboards.ci_group  <----------- Parse test manifest to fetch this value dynamically
```